shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -109,7 +125,14 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI path = new AlluxioURI(args[0]);
     int replication = FileSystemShellUtils.getIntArg(cl, REPLICATION_OPTION, DEFAULT_REPLICATION);
-    distributedLoad(path, replication);
+    Set<String> workerSet = Collections.EMPTY_SET;
+    if (cl.hasOption(WORKER_SET_OPTION.getLongOpt())) {
+      String argOption = cl.getOptionValue(WORKER_SET_OPTION.getLongOpt());
+      String[] workerArray = StringUtils.split(argOption, ",");
+      workerSet = new HashSet<>();
+      Collections.addAll(workerSet, workerArray);
```

Comment:
Is it possible that this adds empty strings, or can hostnames have whitespace around them?